target/linux/octeon/image/Makefile

```diff
@@ -94,4 +94,23 @@ define Device/ubnt_unifi-usg
 endef
 TARGET_DEVICES += ubnt_unifi-usg
 
+define Device/n821
+  DEVICE_VENDOR := Cisco Viptela
+  DEVICE_MODEL := vEdge 1000
+  BOARD_NAME := imt,n821
```

Comment:
this doesn't match the compatible of your dts, does sysupgrade even accept the image?